Wiki.md

```diff
@@ -84,6 +85,21 @@ Then for every new contribution (including your first), you should follow these
 
 9. Once reviewers have approved, the welfare team will merge your PR, and your changes will be automatically deployed to the Oppia wiki. Congratulations!
 
+### Run pylint on python files in the [Oppia-docs-repo](https://github.com/oppia/oppia-web-developer-docs).
+steps:
+
+  1.  Install Pylint
+
+      If pip refers to python3, then run:
+      > pip install pylint
```

Comment:
Can you make these code blocks instead of block quotes? ditto below

Reply:
Done